book.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+let myLibrary = [];
+
+function Book(author, title, pageCount, hasRead) {
+  const book = {
+    author,
+    title,
+    pageCount,
+    hasRead,
+  };
+  return book;
+}
+
+function clearDisplay() {
+  const bookholder = document.getElementById('bookholder');
+  bookholder.innerHTML = '';
+}
+
+function displayLibrary() {
+  clearDisplay();
+}
+
+function addBookToLibrary() {
+  const author = document.getElementById('author').value;
+  const title = document.getElementById('title').value;
+  const pageCount = document.getElementById('pageCount').value;
+  const hasRead = document.getElementById('check').checked;
+  const newBook = Book(author, title, pageCount, hasRead);
+  myLibrary.push(newBook);
+  displayLibrary();
+
+  const savedLibrary = JSON.stringify(myLibrary);
+  localStorage.setItem('book', savedLibrary);
+  // const book =
+  localStorage.getItem('book');
+  localStorage.clear();
+}
+
+function removeBookFromLibrary(bookTitle) {
+  const newLibrary = myLibrary.filter((book) => book.title !== bookTitle);
+  myLibrary = newLibrary;
+  displayLibrary();
+}
```

Comment:
- [ ] <b>REQUIRED:</b> The `Remove` button is removing all books instead of removing just one book. Check out this [Video](https://www.loom.com/share/12809e1f7146466f844be60d585b4ca8) in case you want to see what happens.

Reply:
In your video, you basically duplicated the same book, bro. That's why all of them got deleted. If you changed the title, then it would work normally.